CONTRIBUTING.md

```diff
@@ -535,6 +535,32 @@ Refer datalad/config.py for information on how to add these environment variable
 
 # Release(s) workflow
 
+## Branches
+
+- `master`: changes toward the next `MAJOR.MINOR.0` release (e.g., `0.13` as of time of writing).
```

Comment:
I think the "(e.g., ...)" bit can safely be dropped (same for the instance below).  If it stays, this example tag should have a matching format by appending a ".0".

Reply:
was dropped